Game/Source/GDKTestGyms/Private/Task_DelaySimulated.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Task_DelaySimulated.h"
+
+#include "Engine/World.h"
+#include "GameplayTasksComponent.h"
+#include "Kismet/KismetSystemLibrary.h"
+#include "TimerManager.h"
+
+#include "EngineClasses/SpatialGameInstance.h"
+
+UTask_DelaySimulated::UTask_DelaySimulated(const FObjectInitializer& ObjectInitializer)
+	: Super(ObjectInitializer)
+{
+	bTickingTask = true;
+	bSimulatedTask = true;
+}
+
+void UTask_DelaySimulated::Activate()
+{
+	UWorld* World = GetWorld();
+	FTimerHandle TimerHandle;
+	World->GetTimerManager().SetTimer(TimerHandle, this, &UTask_DelaySimulated::OnFinish, 1.0f, false);
+
+	if (USpatialGameInstance* GameInstance = Cast<USpatialGameInstance>(World->GetGameInstance()))
```

Comment:
What's this for?

Reply:
Ah, I was previously printing a log with `UE_LOG` so I needed it to get the WorkerId, I'll remove this now.